sim/midas/src/main/scala/midas/passes/fame/FAMETransform.scala

```diff
@@ -147,34 +147,63 @@ object FAMEModuleTransformer {
      */
     val nReset = DoPrim(PrimOps.Not, Seq(WRef(hostReset)), Seq.empty, BoolType)
 
+    /*
+     * At simulation time zero do a clock low for all clock domains: let all
+     * combinational paths resolve based on initial tokens and register values.
+     * We can spoof this this by setting clockEnable for all domains without
+     * actually ungating those clocks.
+     */
+    val doneInit = hostFlagReg(ns.newName("doneInit"), resetVal = UIntLiteral(0))
+    val doneInitConnect = Connect(NoInfo, WRef(doneInit), Or(WRef(doneInit), WRef(finishing)))
+
+    case class TargetClockMetadata(
+      targetSourcePort: Port,
+      // I'm not convinced these are the right names for this...
```

Comment:
I'm not convinced on the names either.  Are `clockLow` and `clockHigh` `negedge` and `posedge` with separate gating enable signals for a single target clock domain or is it something completely different?